test/ui_test_suite.rb

```diff
@@ -2373,9 +2383,10 @@ def teardown
     assert rendered, "cluster plot did not finish rendering, expected true but found #{rendered}"
 
     # check for coordinate labels
-    labels = @driver.execute_script("return layout.scene.annotations;")
-    assert_not_nil labels, 'Did not return coordinate labels'
-    assert labels.size == 8, "Did not find coorect number of coordinate labels, expected 8 but found #{labels.size}"
+    # TODO: figure out how to get these back into first create-study: test so we can assert this
```

Comment:
what's the SCP team norm around committing TODOs?  My preferred approach is that TODOs require creation of a corresponding ticket, and then the TODO has to reference the ticket -- otherwise the comment should use more neutral language like 'MAYBE' or 'Would be nice if'.  But I'll defer to whatever SCP usually does

Reply:
We don't have a team norm related to TODOs in code that I'm aware of.  Also there was no ticket relating to this, but I just created it ([SCP-2072](https://broadworkbench.atlassian.net/browse/SCP-2072)).  These files are rarely uploaded by SCP users, so I considered this minor regression to be of low priority.  But I can at least put in a reference to the ticket for this.